root/inc/notification.html

```diff
@@ -24,6 +24,17 @@
             text   = "The maintainer of this distribution is looking for someone to take over!",
             body   = "Hello,\r\n\r\nI'm interested in taking over the %s module. If you can grant me the privilege I will release the next version.\r\n\r\n"
         },
+        DEPRECATED = {
```

Comment:
The content of this section looks wrong.

Reply:
Wrong in what sense?  Similar to what @Grinnz had commented on, in that we do not need to provide an indication that the author may be looking for someone to take over the distribution?